### ISSUE TITLE
Is there a process for capturing relevant discussion from Slack into documentation?

### ISSUE THREAD

**Issue by nerdstein** [GetDKAN/ops#93]
Since DKAN slack is not a paid Slack, it does not have an archive. This limits the use of things like pinned items, which can help to persistently track useful items.

One other option is to consider a process in which relevant Slack conversations are subsequently tracked within DKAN's github documentation.

Are there any thoughts on how this could be done?

**Comment by dafeder** [comment]
How about we start capturing the archives in https://slackarchive.io/? Looks like this is actually better for a public/community slack; we'll get SEO on the conversations.

**Comment by lukefretwell** [comment]
@nerdstein @nerdstein somewhat related: https://github.com/GetDKAN/ops/issues/52

When I saw the help request this morning, I thought the same as @nerdstein. Would be good to be more diligent about documenting these, perhaps directly into DKAN repo with 'Docs' label. @janette @stefaniefgray something to think about. I've added a 'Docs' label to this repo and tagged this issue for future docs meetings discussion.
